### ISSUE TITLE
White screen on iOS

### ISSUE THREAD

**Issue by vstepaniuk** [standardnotes/forum#2311]
Am I the only one who gets a white screen bug on iOS for more than a week already? Maybe I have some unique settings, but it is still a severe bug

**Describe the bug**

You can enter the passcode:

![5DD9E441-673A-44F9-9758-D2A78F2031FD](https://user-images.githubusercontent.com/24701640/204127744-2b1a2794-d636-477e-b870-3ee914ad525d.png)

But then you just get a white screen

![8B997DFF-7CB9-4579-B254-0BE05CA44DAC](https://user-images.githubusercontent.com/24701640/204127751-29fe11a4-2034-4dc6-b04c-8481e819e808.png)

**To Reproduce**
1. Close the app
2. Open the app
3. Enter passcode
4. Get the white screen

**Smartphone (please complete the following information):**
 - Device: iPhone 8
 - OS: iOS 15.6.1
 - SN app version: 3.46.28

**Comment by mike-to-spec** [comment]
I see the same on iPhone 12 with iOS 16.1.1.

**Comment by jacobgkau** [comment]
This issue may be a duplicate of one already reported, since you have a passcode enabled: https://github.com/standardnotes/forum/issues/2309

**Comment by atmoio** [comment]
Does this happen consistently or randomly?

**Comment by vstepaniuk** [comment]
Every time

**Comment by atmoio** [comment]
Can you try selecting Switch workspace > Add another workspace, and see if that allows the application to load? Then sign into your account in that workspace and see if you observe different launch behavior.

**Comment by vstepaniuk** [comment]
same

**Comment by Pistoleer** [comment]
I get the same thing, so you are not alone. Rarely I get a search screen, but then everything goes white.

**Comment by feinersaft** [comment]
I can also confirm this behaviour on iOS 16.1.1 using a passcode.

**Comment by Pistoleer** [comment]
So ... my troubleshooting ...
Uninstall and reinstall. No help.
Soft reset of iphone, uninstall and reinstall. No help.
Uninstall. Soft reset of iphone. Reinstall. No help.

Please work on a quick fix!
![1](https://user-images.githubusercontent.com/92338217/204611526-42a980ba-1e9a-4011-9a12-1dc21e8dc6db.jpeg)

**Comment by homberger** [comment]
Makes notes unusable on iPhone as I am not able to search. I am using a passcode for protected notes, too.

**Comment by honn13** [comment]
Similar issue here. The only difference is that my white screen appeared after opening a note and editing it, when I clicked "Done" on the iPhone keyboard, the screen turned white. This issue has been around for weeks, even after multiple updates. iPhone 13 Mini, iOS 16.1.2.

**Comment by feinersaft** [comment]
> Similar issue here. The only difference is that my white screen appeared after opening a note and editing it, when I clicked "Done" on the iPhone keyboard, the screen turned white. This issue has been around for weeks, even after multiple updates. iPhone 13 Mini, iOS 16.1.2.

True, I also encountered it once after editing a note.

**Comment by homberger** [comment]
Is everyone using an iPhone with a small display?

**Comment by effieeee** [comment]
Fix for white screens has been pushed on [v3.127.2](https://github.com/standardnotes/app/commit/91f3295cffd54c7a5de01633add9b6abdb769cdf), for iOS, update is still pending on the App Store! 🙏

**Comment by Pistoleer** [comment]
Thanks!On Dec 20, 2022, at 06:55, Effie Beldia ***@***.***> wrote:﻿
Fix for white screens has been pushed on v3.127.2, for iOS, update is still pending on the App Store! 🙏

**Comment by honn13** [comment]
I have just installed the said update (version 3.127.5) on my iPhone 13 Mini, but the blank screen issue still remains. It has been weeks, this issue is frustratingly devastating...

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by homberger** [comment]
I was able to fix this problem by changing some settings of my iPhone! It was just luck I guess. This also fixed a problem with another app (Rocket.Chat) on my phone. 🥳

Settings  -> Accessibility -> Motion

I disabled and then reenabled "Reduce Motion". I also changed the dynamic text size to smaller and then back to default. After that the problem of white screens disappeared and I was not able to change back the settings to any values so that the white screens reappear.

I did not update to the most recent version (3.127.5) of the Standard Notes App which should fix the black/white screens.

This needs further research for sure, but for me everything is working fine finally. And I do hesitate to install the latest update of the app...

iOS 16.1.2
iPhone SE 2020 (MX9T2ZD/A)
Standard Notes App v3.125.0 (401)

Link to the Rocket.Chat issue:
https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/3846#issuecomment-1360850353

**Comment by honn13** [comment]
Thanks for this. I tried playing around with the Motion settings and in it I have isolated the issue to “Prefer Cross-Fade Transitions”. When that feature is on, the white screen issue is reproduced. For now I will turn it off until a fix is issued.

Best.

**Comment by effieeee** [comment]
@homberger Very intriguing, thank you so much for sharing this 🙏Hopefully it helps the devs further locate the root cause of the white screen issues!

**Comment by effieeee** [comment]
@homberger Thank you very much for sharing this! So far the issue does appear to be linked to the animations, so reducing the motion settings could also help as a workaround, though the devs will still look into what else can be adjusted to resolve this 🙏

**Comment by myreli** [comment]
I will close this issue so we can keep track here: https://github.com/standardnotes/forum/discussions/2906. 

For now, the workaround (motion settings) is still recommended until a fix is found, and we will update that thread.
